tests/Report/DevPreview-ex1-a.html

```diff
@@ -78,9 +78,9 @@ <h1>Preview perspective of 4.6.32</h1>
 <h1>Preview perspective of 4.5.10</h1>
 <table><tbody>
 <tr><td valign='top'><code>4.5.10:upgrade:name</code></td><td valign='top'>upgrade</td></tr>
-<tr><td valign='top'><code>4.5.10:upgrade:severity</code></td><td valign='top'>warning</td></tr>
-<tr><td valign='top'><code>4.5.10:upgrade:title</code></td><td valign='top'>CiviCRM Version End-of-Life</td></tr>
-<tr><td valign='top'><code>4.5.10:upgrade:message</code></td><td valign='top'><p>CiviCRM 4.5 has reached its end-of-life. Security updates are not provided anymore. Please upgrade to the latest stable release (5.0) or an extended-security release (5.7).</p><p><strong>Release history</strong></p><a href="https://download.civicrm.org/about/4.6.0" target="_blank">4.6.0</a> was released on 2015-04-03. The latest patch revision is <a href="https://download.civicrm.org/about/4.6.36" target="_blank">4.6.36</a> (2018-03-07).<br/><a href="https://download.civicrm.org/about/4.7.0" target="_blank">4.7.0</a> was released on 2016-01-27. The latest patch revision is <a href="https://download.civicrm.org/about/4.7.32" target="_blank">4.7.32</a> (2018-04-20).<br/><a href="https://download.civicrm.org/about/5.0.0" target="_blank">5.0.0</a> was released on 2018-04-04.<br/></td></tr>
+<tr><td valign='top'><code>4.5.10:upgrade:severity</code></td><td valign='top'>critical</td></tr>
+<tr><td valign='top'><code>4.5.10:upgrade:title</code></td><td valign='top'>CiviCRM Security Upgrade Needed</td></tr>
+<tr><td valign='top'><code>4.5.10:upgrade:message</code></td><td valign='top'><p>CiviCRM 4.5 has reached its end of life. Security updates are not provided anymore. Please upgrade to the latest stable release (5.0) or an extended-security release (5.7).</p><p><strong>Release history</strong></p><a href="https://download.civicrm.org/about/4.6.0" target="_blank">4.6.0</a> was released on 2015-04-03. The latest patch revision is <a href="https://download.civicrm.org/about/4.6.36" target="_blank">4.6.36</a> (2018-03-07).<br/><a href="https://download.civicrm.org/about/4.7.0" target="_blank">4.7.0</a> was released on 2016-01-27. The latest patch revision is <a href="https://download.civicrm.org/about/4.7.32" target="_blank">4.7.32</a> (2018-04-20).<br/><a href="https://download.civicrm.org/about/5.0.0" target="_blank">5.0.0</a> was released on 2018-04-04.<br/></td></tr>
```

Comment:
@agh1 this string is a bit dated -but since it's in 'tests' that may not matter

 5.0 is not latest, 4.6 is gone & the ESR is 5.13 (5.7 is also still current but wouldn't encourage anyone to actively adopt it if not already on it)

> Please upgrade to the latest stable release (5.0) or an extended-security release (5.7).</p><p><strong>Release history</strong></p><a href="https://download.civicrm.org/about/4.6.0" target="_blank">4.6.0</a> was released on 2015-04-03. The latest patch revision is <a href="https://download.civicrm.org/about/4.6.36" target="_blank">4.6.36</a> (2018-03-07).<br/><a href="https://download.civicrm.org/about/4.7.0" target="_blank">4.7.0</a> was released on 2016-01-27. The latest patch revision is <a href="https://download.civicrm.org/about/4.7.32" target="_blank">4.7.32</a> (2018-04-20).<br/><a href="https://download.civicrm.org/about/5.0.0" target="_blank">5.0.0</a> was released on 2018-04-04.<br/></td></tr>

Reply:
Yeah these tests are based upon the test files, not the current JSON:
https://github.com/civicrm/civicrm-pingback/blob/304f156c7dffdb691353a1af2d76c6d25e79d30b/tests/ex1.json#L203-L211